homeassistant/components/sensor/nmbs.py

```diff
@@ -0,0 +1,199 @@
+"""
+Get ride details and liveboard details for NMBS (Belgian railway).
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.nmbs/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, ATTR_ATTRIBUTION
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'NMBS'
+DEFAULT_NAME_LIVE = "NMBS Live"
+
+DEFAULT_ICON = 'mdi:train'
+DEFAULT_ICON_ALERT = 'mdi:alert-octagon'
+
+CONF_STATION_FROM = 'station_from'
+CONF_STATION_TO = 'station_to'
+CONF_STATION_LIVE = 'station_live'
+
+REQUIREMENTS = ['pyrail==0.0.3']
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_STATION_FROM): cv.string,
+    vol.Required(CONF_STATION_TO): cv.string,
+    vol.Optional(CONF_STATION_LIVE, default=""): cv.string,
```

Comment:
Looks like that the default is only used to check the variable on line 72. I suggest to just go with `None`.

Reply:
Ah, I put an empty string here because I thought I'd read it on developers.home-assistant.io somewhere that you should use empty strings instead. Will change it to None.